ddtrace/opentelemetry/_span.py

```diff
@@ -216,14 +216,27 @@ def set_status(self, status, description=None):
     def record_exception(self, exception, attributes=None, timestamp=None, escaped=False):
         # type: (BaseException, Optional[Attributes], Optional[int], bool) -> None
         """
-        Records the type, message, and traceback of an exception as Span attributes.
-        Note - Span Events are not currently used to record exception info.
+        Records an exception as an event
         """
         if not self.is_recording():
             return
+        # Set the error type, error message and error stacktrace tags on the span
         self._ddspan._set_exc_tags(type(exception), exception, exception.__traceback__)
+        if attributes and "exception.message" in attributes:
+            # Error message stored in attributes must take precedence over the exception message
+            self.set_attribute(ERROR_MSG, attributes["exception.message"])
```

Comment:
Does the override work for all of the error tracking fields?
- `error.msg` (attributes["exception.message"] overrides)
- `error.type` (attributes["exception.type"] overrides)
- `error.stack` (attributes["exception.stacktrace"] overrides)

Reply:
Not yet. I will make this change